app/views/partials/social/_social-share.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="social--media">
   <% t('social.share').keys.map(&:to_s).each do |social_medium| %>
-    <%= link_to '', t("social.share.#{social_medium}.url", url: social_medium ),
+    <%= link_to '', t("social.share.#{social_medium}.url", url: url_encode(request.base_url) ),
```

Comment:
@stanleypliu `request.base_url` won't cover the intent of sharing a specific resource page but will always refer to the home page regardless. I guess this needs changing

Reply:
Apologies, at the time of making this PR I wasn't sure as to whether we wanted to share the base website or the current page, and then I forgot to amend this. Good catch